src/main/java/repository/OrderRepository.java

```diff
@@ -126,10 +149,89 @@ public List<Order> getOrdersBetweenDates(Date from, Date to) { //Right condition
     }
 
     public List<Order> getUserOrdersBetweenDates(UUID userId, Date from, Date to) {
-        return null;
+        String GET_USER_ORDERS_BETWEEN_DATES = "SELECT * FROM \"order\" WHERE client_id = ? AND (\"from\" >= ? AND \"to\" <= ?) AND status";
+        try (PreparedStatement statement = connection.prepareStatement(GET_USER_ORDERS_BETWEEN_DATES)) {
+
+            statement.setObject(1, userId);
+            statement.setDate(2, DateTimeUtil.toSqlDate(from));
+            statement.setDate(3, DateTimeUtil.toSqlDate(to));
+
+            ResultSet resultSet = statement.executeQuery(GET_USER_ORDERS_BETWEEN_DATES);
+
+            List<Order> list = new ArrayList<>();
+            while (resultSet.next()) {
+                Order order = (Order) Mapper.mapSingleFromResultSet(resultSet, Order.class);
+                list.add(order);
+            }
+
+            resultSet.close();
+
+            return list;
+        } catch (SQLException exception) {
+            log.error("Can not process statement", exception);
+            throw new CarRentalException(exception.getMessage());
+        }
+    }
+
+    public boolean isExistById(UUID id) {
+        String IS_EXIST = "SELECT EXISTS (SELECT 1 FROM \"order\" WHERE id=? AND status);";
+        try (PreparedStatement statement = connection.prepareStatement(IS_EXIST)) {
+            statement.setObject(1, id);
+
+            ResultSet resultSet = statement.executeQuery();
+
+            boolean isExist = Boolean.FALSE;
+            if (resultSet.next()) {
+                isExist = resultSet.getBoolean(1);
+            }
+
+            resultSet.close();
+            return isExist;
+        } catch (SQLException exception) {
+            log.error("Can not process statement", exception);
+            throw new CarRentalException(exception.getMessage());
+        }
+    }
+
+
+    public boolean isUserExistById(UUID userId) {
+        String IS_USER_EXIST = "SELECT EXISTS (SELECT 1 FROM \"order\" WHERE client_id=? AND status)";
+        try (PreparedStatement statement = connection.prepareStatement(IS_USER_EXIST)) {
+            statement.setObject(1, userId);
+
+            ResultSet resultSet = statement.executeQuery();
+
+            boolean isExist = Boolean.FALSE;
+            if (resultSet.next()) {
+                isExist = resultSet.getBoolean(1);
+            }
+
+            resultSet.close();
+            return isExist;
+        } catch (SQLException exception) {
+            log.error("Can not process statement", exception);
+            throw new CarRentalException(exception.getMessage());
+        }
     }
 
-    private Timestamp getTimestamp(java.util.Date date) {
-        return date == null ? null : new java.sql.Timestamp(date.getTime());
+
+    public boolean isCarExistById(UUID carId) {
```

Comment:
Could you please rename this method. We want to check if exists an order by car id, but not a car